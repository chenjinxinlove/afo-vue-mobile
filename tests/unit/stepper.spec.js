import { mount } from '@vue/test-utils'
import Demo from '../../src/stepper/demo'
import Vue from 'vue'
import Stepper from '../../src/stepper'

describe('Rate.vue', () => {
  // 测试是否可以被使用
  it('use', () => {
    Vue.use(Stepper)
    expect(Vue.component(Stepper.name)).toHaveProperty('name', 'VueComponent')
  })
  // demo测试快照
  it('demo snapshot', () => {
    expect(mount(Demo)).toMatchSnapshot()
  })
})