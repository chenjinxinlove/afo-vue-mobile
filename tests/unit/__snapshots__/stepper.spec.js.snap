// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rate.vue demo snapshot 1`] = `
<div>
  <p>基础用法</p>
  <div class="afo-stepper">
    <button class="afo-stepper__minus afo-stepper__minus--disabled"></button>
    <input type="number" class="afo-stepper__input">
    <button class="afo-stepper__plus"></button>
  </div>
  <div style="height: 20px;"></div>
  <p>禁用状态</p>
  <div class="afo-stepper">
    <button class="afo-stepper__minus afo-stepper__minus--disabled"></button>
    <input type="number" disabled="disabled" class="afo-stepper__input">
    <button class="afo-stepper__plus afo-stepper__plus--disabled"></button>
  </div>
  <div style="height: 20px;"></div>
  <p>高级用法</p>
  <div class="afo-stepper">
    <button class="afo-stepper__minus"></button>
    <input type="number" class="afo-stepper__input">
    <button class="afo-stepper__plus"></button>
  </div>
</div>
`;
